bigbluebutton/greenlight
# (feat) Add access code to Rooms

## Description

Add AccessCode to Rooms.
If an access code is generated by the Room admin, the User will be prompted to enter the access code before being able to join a room.

## Testing Steps


- Log in your GLv3 profile.
- Go into a Room, and click on the Copy button to generate a "Join Room URL".
- Open an incognito browser window.
- In the incognito window, navigate to the "Join Room URL".
- Switch back to the non-incognito window. 
- Go to Room/Room Settings and play with the Generate/Delete access code feature.
- Generating an access code should now instantly prompt the incognito User to enter the access code.

## Screenshots (if appropriate):

![image](https://user-images.githubusercontent.com/43917914/172940484-a97987c2-2572-40f1-8532-585964ba36c9.png)


## TODO
- Verify Code Generation design w/ Tyler
